zonemaster/zonemaster-engine
# Automatically set header fields in new PO files

## Purpose

Automatically update fields in new PO file with the following values:

```
"Project-Id-Version: 1.0.0\n"
"Language-Team: Zonemaster Team\n"
"Content-Type: text/plain; charset=UTF-8\n"
```

## Context

Continuity of #1185 and addresses https://github.com/zonemaster/zonemaster-engine/pull/1185#pullrequestreview-1281532445

## Changes

`new-po` rule in share/GNUMakefile.

## How to test this PR

Run `make POLANG=xx new-po` and check that the fields are automatically set with the default values.
